app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :user_id
+  has_many :comments, foreign_key: :user_id
+  has_many :likes, foreign_key: :user_id
```

Comment:
Please add `author_id` as a foreign key as can be seen in [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). The goal here is to set up associations between models correctly according to the given ERD diagram.